api/v1alpha3/azurecluster_validation.go

```diff
@@ -184,3 +203,64 @@ func validateIngressRule(ingressRule *IngressRule, fldPath *field.Path) *field.E
 
 	return nil
 }
+
+func validateAPIServerLB(lb LoadBalancerSpec, old LoadBalancerSpec, cidrs []string, fldPath *field.Path) field.ErrorList {
+	var allErrs field.ErrorList
+	// SKU should be Standard and is immutable.
+	if lb.SKU != SKUStandard {
```

Comment:
if we only support standard we should remove https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/3c5cb5b975548137d1675bf685766d624bcfa48b/api/v1alpha3/types.go#L174-L175

Do we even need the field in the spec? it's hardcoded in the service right now: https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/cloud/services/loadbalancers/loadbalancers.go#L78

Reply:
we technically don't, I added it as mostly informative and with the possibility that we might support more than one later in mind

good catch though, I should definitely use it if I keep it 😬